{{cookiecutter.folder_name}}/.metadata.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Ah, here we go - I already added this line to the README: 
"make with no arguments compiles the latex using the default_format stored in .metadata.json"